pimd/pim6_cmd.c

```diff
@@ -476,6 +476,47 @@ DEFPY (no_ipv6_pim_rp_prefix_list,
 	return pim_process_no_rp_plist_cmd(vty, rp_str, plist);
 }
 
+DEFPY (ipv6_pim_bsm,
+       ipv6_pim_bsm_cmd,
+       "ipv6 pim bsm",
+       IPV6_STR
+       PIM_STR
+       "Enables BSM support on the interface\n")
```

Comment:
I think `Enable`?

Reply:
Keeping similar as pimV4

Reply:
If it's wrong, better to fix it. Anyways it's new code.

Reply:
Addressed